src/dirs.js

```diff
@@ -21,11 +21,9 @@ function _isStackIndex(index) {
 
 function _parseStackIndex(index) {
   if (_isStackIndex(index)) {
-    if (Math.abs(index) < _dirStack.length + 1) { // +1 for pwd
+    if (Math.abs(index) < _dirStack.length + 1) // +1 for pwd
```

Comment:
Nit: Can we either add curly braces here, or have the `return` on the same line as the `if`?